rector.php

```diff
@@ -24,12 +26,17 @@
 
     $rectorConfig->sets([
         LevelSetList::UP_TO_PHP_74,
+        PHPUnitSetList::PHPUNIT_90,
+        PHPUnitSetList::PHPUNIT_91,
```

Comment:
Isn't there a `UP_TO_*` ?

Reply:
Good catch! 

https://github.com/rectorphp/rector-phpunit/blob/main/src/Set/PHPUnitLevelSetList.php

I'll update this later